plugins/lookup/filetree.py

```diff
@@ -55,6 +59,10 @@
 - name: list all files under web/
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.filetree', 'web/') }}"
+
+- name: list all files under web/ exclude .git and .node_modules
+  ansible.builtin.debug:
+    msg: "{{ lookup('community.general.filetree', 'web/', exclude='.*(.git|.node_modules).*') }}"
```

Comment:
If you really want to exclude `.git` and `.node_modules`, you should write:
```suggestion
    msg: "{{ lookup('community.general.filetree', 'web/', exclude='^(\.git|\.node_modules)$') }}"
```
since the regular expression is applied to every individual file or directory name (and not to the path).